python/ray/dag/experimental/types.py

```diff
@@ -0,0 +1,109 @@
+from typing import Tuple
+
+import numpy as np
+import torch
+
+import ray.util.serialization
+from ray.util.annotations import DeveloperAPI, PublicAPI
+
+
+class DAGNodeOutputType:
+    pass
+
+
+@DeveloperAPI
+def do_register_custom_dag_serializers(self):
```

Comment:
```suggestion
def do_register_custom_dag_serializers(self):
```

Can you change the variable name & add types here?